java/src/jmri/InstanceManager.java

```diff
@@ -422,6 +422,20 @@
         return !l.isEmpty();
     }
 
+    /**
+     * Check if a particular type has been initialized without
+     * triggering an automatic initialization.
+     *
+     * @param <T>  The type of the class
+     * @param type The class type
+     * @return true if an item is available as a default for the given type;
+     *         false otherwise
+     */
+    public static <T> boolean isInitialized(@Nonnull Class<T> type) {
```

Comment:
Should this method and the method preceding it (`containsDefault`) not be the same method?

Reply:
Probably.  I think I managed to confuse myself with the multiple meanings of "default" here.

`containsDefault` creates a `List<T>` and fires a property event, but the list is empty so there's no default item.